warehouse/common/src/main/java/datawave/common/cl/OptionBuilder.java

```diff
@@ -19,8 +19,8 @@
     /** Whether or not the next Option's argument is optional; defaults to false. */
     public boolean optionalArg;
     
-    /** The next Option's type; defaults to null. */
-    public Object type;
+    /** The next Option's type; defaults to {@link String#getClass()}. */
+    public Class<?> type = String.class;
```

Comment:
why are we changing the default here?

Reply:
Sorry, I couldn't recall the flagged issue....tried to revert back and reproduce the finding but couldn't reproduce it...which is when I realized this one was actually a "Medium" threshold, not "High" as advertised by this PR. So, this one slipped thru, as it was one of the first I encountered b4 raising to High. My bad

At any rate, I now see that the finding was....

```
Field only ever set to null: datawave.common.cl.OptionBuilder.type [datawave.common.cl.OptionBuilder] At OptionBuilder.java:[line 55] UWF_NULL_FIELD
```
So,  I decided the easiest and safest remedy was simply to mirror the corresponding default setup for org.apache.commons.cli.Option.type. I.e., ...
```
public class Option implements Cloneable, Serializable {
   ....
   private Class<?> type = String.class;
   ....
}
```